go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/operator-framework/helm-operator-plugins
 
-go 1.22.5
+go 1.22.1
+
+toolchain go1.22.5
```

Comment:
I also wonder why is it necessary to specify the toolchain here at all? Is it a declaration that the code compiles with this version? Or that it must be compiled with at least this version? Since the .5 just means some bugfixes according to semver, I wonder why it's important at all 🤷🏻

Reply:
Fair call - removed it